src-docs/src/views/form_controls/form_controls_example.js

```diff
@@ -176,34 +164,52 @@ const switchSnippet = [
 
 import PrependAppend from './prepend_append';
 const PrependAppendSource = require('!!raw-loader!./prepend_append');
-const PrependAppendHtml = renderToHtml(PrependAppend);
 
 import Fieldset from './fieldset';
 const fieldsetSource = require('!!raw-loader!./fieldset');
-const fieldsetHtml = renderToHtml(Fieldset);
 
 import FormControlLayout from './form_control_layout';
 const formControlLayoutSource = require('!!raw-loader!./form_control_layout');
-const formControlLayoutHtml = renderToHtml(FormControlLayout);
 
 import FormControlLayoutRange from './form_control_layout_range';
 const formControlLayoutRangeSource = require('!!raw-loader!./form_control_layout_range');
-const formControlLayoutRangeHtml = renderToHtml(FormControlLayoutRange);
 
 export const FormControlsExample = {
   title: 'Form controls',
   sections: [
     {
       title: 'Text field',
+      text: (
+        <>
+          <p>
+            A simple wrapper around{' '}
+            <EuiCode language="html">{'<input type="text" >'}</EuiCode>.
+          </p>
+          <EuiCallOut>
+            <EuiText size="m">
+              <p>
+                <strong>EuiTextField</strong> is <strong>currently</strong> the
+                only field component to accept{' '}
+                <Link to="/forms/form-layouts#form-and-form-rows">
+                  <strong>EuiFormRow</strong>
+                </Link>{' '}
+                props directly. For instance, when providing the{' '}
+                <EuiCode>label</EuiCode> prop, it will automically wrap itself
+                in a <strong>EuiFormRow</strong>. Other promoted props include{' '}
+                <EuiCode>helpText</EuiCode>, <EuiCode>error</EuiCode> , and{' '}
+                <EuiCode>display</EuiCode>. Providing form row props at this
```

Comment:
Why callout these props and not others? Makes it seem like others might not work.

Reply:
Probably could just use some editing, but the reason is there's a looong list of form row props. And really, I was just trying to give examples of common props. WOuld something like this be more clear?
```suggestion
                in a <strong>EuiFormRow</strong>. Examples of promoted props include{' '}
                <EuiCode>helpText</EuiCode>, <EuiCode>error</EuiCode> , and{' '}
                <EuiCode>display</EuiCode>. Providing form row props at this
```

Reply:
Yup, that LGTM!